app/src/main/java/com/example/playlistmaker/Player/data/PlayerInteractorRepositoryImpl.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.playlistmaker.Player.data
+
+
+import android.media.MediaPlayer
+import com.example.playlistmaker.Player.domain.PlayerInteractorRepository
+import com.example.playlistmaker.Player.domain.PlayerState
+
+
+
+class PlayerInteractorRepositoryImpl: PlayerInteractorRepository {
```

Comment:
🍏 Надо все-таки определиться: Interactor или Repository)  Вообще это не то и не другое, поэтому рекомендую назвать просто:
```suggestion
class PlayerImpl: Player {
```